node/src/server-modules/GlideFtOptions.ts

```diff
@@ -118,3 +118,39 @@ export interface FtCreateOptions {
     /** The prefix of the key to be indexed. */
     prefixes?: GlideString[];
 }
+
+/**
+ * Represents the input options to be used in the FT.SEARCH command.
+ * All fields in this class are optional inputs for FT.SEARCH.
+ */
+export interface FTSearchOptions {
+    /**
+     * Add a field to be returned.
+     * @param fieldIdentifier field name to return.
+     * @param alias optional alias for the field name to return.
+     */
+    returnFields?: { fieldIdentifier: GlideString; alias?: GlideString }[];
```

Comment:
alias vs name @prateek-kumar-improving ?

Reply:
It is alias in documentation. This is correct. Over here it is an alias for the identifier.